packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -282,14 +290,23 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
       return null;
     }
     return (
-      <aside
-        class={`mx-auto ${
-          isLoading ? this.loadingClasses : this.contentClasses
-        }`}
-        part="container"
-      >
-        <article>{isLoading ? <TypingLoader /> : this.renderContent()}</article>
-      </aside>
+      <Host>
+        <atomic-generated-answer-feedback-modal
```

Comment:
By rendering the modal like this, inside the generated-answer component, I'm wondering if the modal-trap aspect is working correctly.

That is: When the modal is opened, and you use the keyboard to tab through all the elements in the modal, it should be "trapped" and not focus on elements outside of the modal, in the rest of the page.

Can you confirm this is working correctly ?

Reply:
yep you can tab through the elements in the modal!

Reply:
Yes, one can tab through the modal elements, but the focus trap does not work as focus moves from the "Send feedback" to the "Like" button under the backdrop. When the modal is opened, the focus should cycle only on the modal elements.